README.md

```diff
@@ -14,7 +14,7 @@
 
 1. [Features](#features)
 2. [Disclaimer](#disclaimer)
-3. [Quark and remote PC](#quark-and-remote-pc)
+3. [Quark and remote PC](#quark-and-remote-browsing)

```

Comment:
It'd be more coherent if the table of contents's name was also changed to "Quark and remote browsing"

Reply:
Why would you mark it as resolved, you didn't make the requested change :P

Reply:
Was on mobile, must have misclicked this! 

Update it and committed :)